### DISCUSSION TITLE
Files to include/exclude

### DISCUSSION THREAD

**Discussion by suntong** [goreleaser/goreleaser#2255] [Q&A] [answered]
How to control which files are included/excluded when using goreleaser for release distribution?

Reason I'm asking, was that I looked at the `linux_amd64.tar.gz` files first, which consists of the following files:

```
-rw-r--r-- runner/docker  1064 2021-05-25 14:37 LICENSE
-rw-r--r-- runner/docker  2145 2021-05-25 14:37 README.e.md
-rw-r--r-- runner/docker  2603 2021-05-25 14:37 README.md
-rwxr-xr-x runner/docker 6623232 2021-05-25 14:38 OpenSesame
```

Then I checked the `linux_amd64.deb` file, and notice that it only consists a single executable.

**Comment by caarlos0** [comment, marked as answer]
Its all on the documentation:

- https://goreleaser.com/customization/archive/
- https://goreleaser.com/customization/nfpm
